packages/sources/layer2-sequencer-health/schemas/env.json

```diff
@@ -28,7 +28,7 @@
     },
     "OPTIMISM_HEALTH_ENDPOINT": {
       "type": "string",
-      "default": "https://mainnet-sequencer.optimism.io/health"
+      "default": ""
     },
```

Comment:
If there will be no other endpoint maintained by optimism, it might be better to outright remove the env var and optimism from the networks that support this check

Reply:
That's a good point, the thing is we're not yet sure that Optimism won't provide a replacement in the future.
We've been notified that the current endpoint is being deprecated but I'm hoping the Optimism team might be able to provide an alternative in the future.
Also, it seems that for other chains with no available endpoints, the env var is listed as empty in the schema, so if we still need to do it for Optimism, I can open a new PR to refactor this env var out for the rest of the chains (Arbitrum, Base & Scroll)